Mathlib/Topology/Algebra/Module/Alternating/Topology.lean

```diff
@@ -119,6 +119,11 @@ end UniformAddGroup
 
 variable [TopologicalSpace F] [TopologicalAddGroup F]
 
+instance instTopologicalAddGroup : TopologicalAddGroup (E [⋀^ι]→L[𝕜] F) :=
+  letI := TopologicalAddGroup.toUniformSpace F
+  haveI := comm_topologicalAddGroup_is_uniform (G := F)
+  inferInstance
```

Comment:
I think it's cleaner to use the embedding just below and [Inducing.topologicalAddGroup](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Group/Basic.html#Inducing.topologicalAddGroup)